slsim/Util/param_util.py

```diff
@@ -174,3 +174,32 @@ def transformmatrix_to_pixelscale(tranform_matrix):
     """
     determinant = np.linalg.det(tranform_matrix)
     return np.sqrt(determinant)
+
+
+def average_angular_size(a, b):
+    """Computes average angular size using semi major and minor axis.
+
+    :param a: value of semi major axis in arcsec
+    :param b: value of semi minor axis in arcsec
+    :return: average angular size in radian
```

Comment:
why do you need in radian and not arcsec^2? Also, a size is not radian but steradian